### ISSUE TITLE
FASTESTVPN NEW CONFIG FILES

### ISSUE THREAD

**Issue by bijfy** [haugene/vpn-configs-contrib#33]
## Before creating this request I have:

**REQUIRED**
- [X] Searched for similar [provider requests][pi] and [container issues][ci]
- [X] [Read the documentation][rd], especially the provider, custom provider, troubleshooting section and FAQ
- [X] Tried to add as much relevant information to the request as possible
- [X] Agreed that my request will be closed if I do not follow this template and will remain closed until I complete the template

[pi]: https://github.com/haugene/vpn-configs-contrib/issues
[ci]: https://github.com/haugene/docker-transmission-openvpn/issues
[rd]: https://haugene.github.io/docker-transmission-openvpn/

### Which provider?

**REQUIRED**
```txt
<FASTESTVPN>
```

### Where are the configs?

**REQUIRED**
```txt
<[Fastestvpn_ovpn(2).zip](https://github.com/haugene/vpn-configs-contrib/files/7040894/Fastestvpn_ovpn.2.zip)>
```
### Additional context

*Optional*

``` <[Fastestvpn_ovpn(2).zip](https://github.com/haugene/vpn-configs-contrib/files/7040894/Fastestvpn_ovpn.2.zip) for optional> ```

**Comment by bijfy** [comment]
[Fastestvpn_ovpn(2).zip](https://github.com/haugene/vpn-configs-contrib/files/7040910/Fastestvpn_ovpn.2.zip)

**Comment by pkishino** [comment]
fixed #32
